File: frontend/src/firebase.js
import { initializeApp } from "firebase/app";
import { getAuth } from "firebase/auth";
import { getStorage } from "firebase/storage";

const firebaseConfig = {
    apiKey: "AIzaSyDCxuvCpAfSSmhd7cVS3Wzm3bnd3GoQudQ",
    authDomain: "hackathons-5f2de.firebaseapp.com",
    projectId: "hackathons-5f2de",
    storageBucket: "hackathons-5f2de.firebasestorage.app",
    messagingSenderId: "515532031820",
    appId: "1:515532031820:web:bd9a665daae44dab0231e9",
    measurementId: "G-MDS30FJ11V"
};


const app = initializeApp(firebaseConfig);
export const auth = getAuth(app);
export const storage = getStorage(app);

File: backend/Firebase.py
from firebase_admin import credentials, initialize_app, storage

cred = credentials.Certificate('backend\FirebaseKey.json')
firebase_app = initialize_app(cred)
storage_bucket = storage.bucket('hackathons-5f2de.firebasestorage.app')

def getStorageBucket():
    return storage_bucket
